src/components/formFields/input.tsx

```diff
@@ -7,19 +7,22 @@ export interface IInputProps {
   name: string;
   disabled?: boolean;
   defaultValue?: string;
+  placeholder?: string;
 }
 
 export const InputPassword: React.FC<IInputProps & IReactHookFormProps> = ({
   disabled,
   name,
   validation,
   register,
+  placeholder,
   errors,
 }) => {
   const [showPassword, setShowPassword] = React.useState<boolean>(false);
 
   return (
     <TextField
+      placeholder={placeholder}
```

Comment:
Goes for all of the `placeholder={placeholder}` props: you can add these with `{... {placeholder}}`. See other comment.